readme.md

```diff
@@ -83,6 +83,13 @@ Default: `true`
 
 Overwrite existing files.
 
+##### followSymbolicLinks
+
+Type: `boolean`\
+Default: `false`
+
+Indicates whether to traverse descendants of symbolic link directories when expanding `**` patterns.
```

Comment:
It's not just when expanding patterns though. It's also relevant even you select a single file as source (no globbing) and that file is a symbolic link. Which I don't think is actually working?